docs/Push-Server-Database.md

```diff
@@ -92,25 +92,29 @@ CREATE TABLE push_app_credentials (
     ios_bundle VARCHAR(255),
     ios_environment VARCHAR(32),
     android_private_key BYTEA,
-    android_project_id VARCHAR(255)
+    android_project_id VARCHAR(255),
```

Comment:
Let's remove SQL completely in a different PR. This update just makes it consistent.